src/test/java/org/broadinstitute/hellbender/tools/spark/sv/StructuralVariationDiscoveryPipelineSparkIntegrationTest.java

```diff
@@ -134,8 +138,34 @@ public void testSVDiscoverPipelineRunnableMiniCluster(final StructuralVariationD
             path = new Path(workingDirectory, "fastq");
             argsToBeModified.set(idx+1, path.toUri().toString());
 
-            new IntegrationTestSpec(String.join(" ", argsToBeModified), SVIntegrationTestDataProvider.dummyExpectedFileNames)
-                    .executeTest("testSVDiscoverPipelineRunnableMiniCluster-", this);
+            runCommandLine(argsToBeModified);
+            svDiscoveryVCFEquivalenceTest(vcfOnHDFS, SVIntegrationTestDataProvider.EXPECTED_SIMPLE_DEL_VCF, Arrays.asList("ALIGN_LENGTHS", "CTG_NAMES"), true);
         });
     }
+
+    public static void svDiscoveryVCFEquivalenceTest(final String generatedVCFPath, final String expectedVCFPath, final List<String> attributesToIgnore, final boolean onHDFS) throws Exception{
+
+        VCFFileReader fileReader;
+        CloseableIterator<VariantContext> iterator;
+        final List<VariantContext> actualVcs;
+        if (onHDFS) {
+            final File tempLocalVCF = BaseTest.createTempFile("variants", "vcf"); tempLocalVCF.deleteOnExit();
+            new Path(generatedVCFPath).getFileSystem(new Configuration()).copyToLocalFile(new Path(generatedVCFPath), new Path(tempLocalVCF.getAbsolutePath()));
```

Comment:
I think you can just use `BucketUtils.copyFile` and not have to worry about getting the HDFS `FileSystem` object here.